tests/shell/CMakeLists.txt

```diff
@@ -40,95 +30,88 @@ function (add_scripttest testname)
 
 	foreach (TARGET true false)
 		if (TARGET)
-			#cascading of if is && for poor people
+
+			# cascading of if is && for poor people
 			if (INSTALL_TESTING)
-			if (NOT ${testname_we} STREQUAL "check_formatting")
-			if (NOT ${testname_we} STREQUAL "check_oclint")
-			if (NOT ${testname_we} STREQUAL "check_plugins")
-			if (NOT ${testname_we} STREQUAL "check_gen")
-			if (NOT ${testname_we} STREQUAL "check_external")
-			if (BUILD_FULL)
-				set (KDB_COMMAND "kdb-full")
-			else (BUILD_FULL)
-				if (BUILD_STATIC)
-					set (KDB_COMMAND "kdb-static")
-				else (BUILD_STATIC)
-					set (KDB_COMMAND "kdb")
-				endif (BUILD_STATIC)
-			endif (BUILD_FULL)
-			set(RACE_COMMAND "$KDB race")
-			set(INCLUDE_COMMON "${INCLUDE_COMMON_INSTALLED_FILE}if [ -z \"$KDB\" ]; then KDB=${KDB_COMMAND}; fi")
-			configure_file(
-				"${filename}"
-				"${CMAKE_CURRENT_BINARY_DIR}/${testname}"
-				@ONLY
-				)
-			install (FILES "${CMAKE_CURRENT_BINARY_DIR}/${testname}"
-				DESTINATION ${TARGET_TOOL_EXEC_FOLDER}
-				PERMISSIONS OWNER_READ OWNER_WRITE OWNER_EXECUTE
-				GROUP_READ GROUP_EXECUTE
-				WORLD_READ WORLD_EXECUTE
-				RENAME ${testname_we}
-				)
-			endif (NOT ${testname_we} STREQUAL "check_external")
-			endif (NOT ${testname_we} STREQUAL "check_gen")
-			endif (NOT ${testname_we} STREQUAL "check_plugins")
-			endif (NOT ${testname_we} STREQUAL "check_oclint")
-			endif (NOT ${testname_we} STREQUAL "check_formatting")
-			endif(INSTALL_TESTING)
+				if (NOT ${testname_we} STREQUAL "check_formatting")
```

Comment:
Can you exclude this file? We should rewrite the if and I do not want to mix reformatting with actual changes in the code. In its current form it is too deeply nested.